### PR TITLE
Hi

### PR DESCRIPTION
**Pull request by Tuhin-Patel**
No issues had

### CODE REVIEW

**Review by avascherocman — APPROVED** [review]
Your pr name should summarize your changes :)

**Review by arizeltser — COMMENTED** [review]
Looks like you put your section inside of another section instead of below it. Make sure "</div>" of the previous person is above your first line

File: index.html [hunk 1]
```diff
@@ -7,8 +7,9 @@
   <body>
     <div id="heads-area">
       <div class="person-frame-inner">
-        <img class="ball" src="images/max_gruber.jpg" />
-        <p>Max Gruber</p>
+        <div class="person-frame-inner">
+          <img class="ball" src="images/patel.4014.jpg" />
+          <p>Tuhin Patel</p>
```
Comment (adambricelis) [inline index.html:12]: Please do not delete Max's head. Try adding your code to the end of the file :)

**Review by adambricelis — CHANGES_REQUESTED** [review]

**Comment by adambricelis** [comment]
Please switch to the master branch, pull the latest changes, create another branch, redo your changes, and submit another PR.

**Pull request closed without merging**